server/src/main/java/com/flipkart/varadhi/config/VaradhiOptions.java

```diff
@@ -3,8 +3,15 @@
 import com.flipkart.varadhi.produce.config.ProducerOptions;
 import lombok.Data;
 
+import static com.flipkart.varadhi.Constants.REST_DEFAULTS.*;
+
 @Data
 public class VaradhiOptions {
```

Comment:
slightly weird name "VaradhiOptions" and also too broad. There is already a ServerConfiguration. that having "varadhiOption" does not make too much sense. For me Server is at same level as Varadhi, server just being the common name to refer to it.